tests/test-cases/Group6-VIC-Machine/6-16-Config.robot

```diff
@@ -278,6 +278,9 @@ Configure VCH resources
     Should Contain  ${output}  --memory-reservation=10
     Should Contain  ${output}  --memory-shares=163840
 
+    ${output}=  Run  bin/vic-machine-linux configure --name=%{VCH-NAME} --target=%{TEST_URL}%{TEST_DATACENTER} --thumbprint=%{TEST_THUMBPRINT} --user=%{TEST_USERNAME} --password=%{TEST_PASSWORD} --timeout %{TEST_TIMEOUT} --storage-quota 1000
```

Comment:
This probably belongs in a separate test case; the rest of this test case is testing successful configuration changes.

Reply:
I will add some positive cases to the sub group of "Configure VCH resources" once the VCH inspect config command support storage-quota, so basically it belongs to this category.